rllib/execution/learner_thread.py

```diff
@@ -90,7 +90,7 @@ def step(self) -> Optional[_NextValueNotReady]:
             for pid, results in multi_agent_results.items():
                 learner_info_builder.add_learn_on_batch_results(results, pid)
             self.learner_info = learner_info_builder.finalize()
-            self.weights_updated = True
+            self.policy_ids_updated.extend(list(self.learner_info.keys()))
```

Comment:
How is this being used now? Couldn't find ussage in Impala diff. 

Also doesn't change need to be reflected in the learner thread code for APEX (which uses a different learner thread class IIRC)?

Reply:
You are right. Need to fix the DQN learner thread as well ...